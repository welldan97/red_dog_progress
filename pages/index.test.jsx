import React from 'react';
import renderer from 'react-test-renderer';

import Index from '../pages/index';

it('It renders', () => {
  const component = renderer.create(<Index />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
