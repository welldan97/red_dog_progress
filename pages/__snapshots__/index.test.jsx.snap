// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders 1`] = `
<div>
  <nav
    className="navbar navbar-toggleable navbar-inverse bg-danger"
  >
    <button
      className="navbar-toggler navbar-toggler-right"
      type="button"
    >
      <span
        className="navbar-toggler-icon"
      />
    </button>
    <a
      className="navbar-brand"
      href="/"
    >
      Red Dog Progress
    </a>
    <div
      className="collapse show navbar-collapse"
      style={Object {}}
    >
      <ul
        className="ml-auto navbar-nav"
      >
        <li
          className="nav-item"
        >
          <a
            className="nav-link"
            href="/components/"
            onClick={[Function]}
          />
        </li>
      </ul>
    </div>
  </nav>
  <div
    className="mt-5 container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <h2>
          Your weekly progress:
        </h2>
      </div>
    </div>
  </div>
</div>
`;
